OverlayPlugin.Core/EventSources/MiniParseEventSource.cs

```diff
@@ -29,8 +29,16 @@ partial class MiniParseEventSource : EventSourceBase
             { 23, "LookingForParty" },
         };
 
+        private const string CombatDataEvent = "CombatData";
```

Comment:
Sorry for this extra refactoring, but I just don't like repeating magic strings.